xml/System.IO/Path.xml

```diff
@@ -33,8 +33,6 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The .NET Framework does not support direct access to physical disks through paths that are device names, such as "\\\\.\PHYSICALDRIVE0 ".  
```

Comment:
Would it be better to say something like: "The .NET Framework 4.6.2 and higher versions support direct access to physical disks through paths that are device names, such as "\\\\.\PHYSICALDRIVE0 ".  The .NET Framework 4.6.2 and higher versions allows access to any device path as long as the user has the appropriate rights."?

Reply:
The original comment wasn't very precise. Here is the code in question on desktop: https://referencesource.microsoft.com/#mscorlib/system/io/filestream.cs,728

Really any `\\.\` path was blocked for `FileStream`. The `\\?\` variant was blocked overall via invalid path character checks (the `?` was seen as a wildcard). For compatibility sake with limited trust scenarios I only opened these up under full trust and the new AppContext switch (UseLegacyPathHandling) which is off by default when targeting 4.6.2 or higher. For .NET Core there is no CAS of course, and there are no other checks, they are just always supported.

It would be great to get a doc page that explains the path formats for Windows, I'm just not sure where that would go. I've written it up on my blog here: https://blogs.msdn.microsoft.com/jeremykuhne/2016/04/21/path-format-overview/  Together with the info in the paragraph above it would make navigating this stuff a bit easier I think.

Reply:
@JeremyKuhne, I think it's a great idea. @mairaw and @rpetrusha - I think this would go in the .NET Guide, what do you think?

Reply:
I agree @JRAlexander. Probably under or even on this topic: https://docs.microsoft.com/en-us/dotnet/standard/io/

Reply:
I also agree with you, @JeremyKuhne and @JRAlexander. And I agree with you about its location, @mairaw. I'll open an issue.